lib/galaxy_test/api/test_folder_contents.py

```diff
@@ -11,6 +11,9 @@
 )
 from ._framework import ApiTestCase
 
+OWNNER_USER = 'owner@user.com'
```

Comment:
Misspelled and unused after it is setup right?

Reply:
Absolutely, Thanks for the catch!